packages/editor/src/core/extensions/clipboard.ts

```
@@ -0,0 +1,153 @@
+              }
+            }
+
+
+            if (isTableContent && isWholeSelectionCopied) {
+              return processTableContent(slice.content.firstChild);
+            }
+
+            if (isTableContent && !isWholeSelectionCopied) {
+              const tableNode = slice.content.firstChild;
+
+              if (tableNode.content.firstChild && tableNode.content.firstChild.content.firstChild) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)